src/main/js/components/table/Table.jsx

```diff
@@ -73,12 +80,57 @@ class Table extends React.Component {
     Dispatcher.off(ActionTypes.SHOW_ROW_CONTEXT_MENU, tableContextMenu.showRowContextMenu, this);
     Dispatcher.off(ActionTypes.CLOSE_ROW_CONTEXT_MENU, tableContextMenu.closeRowContextMenu, this);
     Dispatcher.off(ActionTypes.DUPLICATE_ROW, tableRowsWorker.duplicateRow, this);
+    Dispatcher.off(ActionTypes.COPY_CELL_CONTENT, this.setCopyOrigin);
+    Dispatcher.off(ActionTypes.PASTE_CELL_CONTENT, this.pasteCellTo);
 
     window.removeEventListener("resize", this.windowResize);
     this.props.table.rows.off("add", tableRowsWorker.rowAdded.bind(this));
     window.GLOBAL_TABLEAUX.tableRowsDom = null;
   }
 
+  setCopyOrigin = cell => {
+    this.pasteOriginCell = cell;
+  };
+
+  pasteCellTo = ({cell}) => {
+    const src = this.pasteOriginCell.cell;
+    if (cell.kind === "link" && !(cell.column.id === src.column.id)) { // only copy same cell type or links from same column
+      ActionCreator.showToast(<div id="cell-jump-toast">{i18n.t("table:copy_links_error")}</div>, 3000);
+      return;
+    }
+
+    if (!cell.isMultilanguage && contains(cell.kind, ["dateTime", "date", "number"])) { // single-value cell can be copied safely
+      cell.save({value: src.value})
+    } else {
+      ActionCreator.openOverlay({
+        head: <div className="overlay-header">{i18n.t("table:confirm_copy.header")}</div>,
+        body: (
+          <div id="confirm-copy-overlay-content" className="confirmation-overlay">
+            <div className="info-text">{i18n.t("table:confirm_copy.info")}</div>
+          </div>
+        ),
+        footer: (
+          <div className="button-wrapper">
+            <a href="#" className="button positive"
+               onClick={() => {
+                 cell.save({value: src.value});
```

Comment:
if langtags of `cell.value` is different to langtags of `src.value` the "old" langtags should be set to `null`

```
cell.value = {"de": ..., "en": ...}
src.value = {"en": ..., "pl": ...}

// {"de": ..., "en": ...} <- {"de": null, "en": ..., "pl": ...}
```